src/redux/books/books.js

```diff
@@ -1,20 +1,58 @@
-import { createSlice } from '@reduxjs/toolkit';
-import books from './data';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
+import axios from 'axios';
+// Initiale state.
+const initialState = [];
 
-export const booksSlice = createSlice({
-  name: 'books',
-  initialState: { value: books },
-  reducers: {
-    addBook: (state, action) => {
-      state.value.push(action.payload);
-    },
+const api = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/yrhTmLYwYcDLEAHmfCpQ/books';
```

Comment:
# Congratulations:
- Connect their React-Redux application to an existing API.
-  Create and remove books in a remote server. 
- Use Redux Toolkit.
- 
![Capture](https://user-images.githubusercontent.com/63972670/194703568-81fec53b-38f7-4ef1-bf13-43d46ee67883.JPG)